### ISSUE TITLE
Customize API's URL

### ISSUE THREAD

**Issue by takumats** [insight-technology/restful-functions#18]
Now API's URL prefix like 'call' and 'call/blocking' are fixed.
But in some cases, we'd like to customize the URL.

current(fixed):
- http://localhost:8888/call/my_function

customized(examples):
- http://localhost:8888/api/my_function
- http://localhost:8888/api/v1.1/my_function
- http://localhost:8888/customers/new

**Comment by yszkst** [comment]
restful-functions already can set another endpoint.

since v2.0.0 

current (normaily)

```
server.add_function(
        addition,  # Function
        [
            ArgDefinition('x', ArgType.INTEGER, True, 'x'),
            ArgDefinition('y', ArgType.INTEGER, True, 'y'),
        ],  # Args
        1,  # Max Concurrency
        'addition')  # Description
```
 * http://localhost:8888/my_function  

and, you can set another endpoint with fifth parameter.


```
server.add_function(
        addition,  # Function
        [
            ArgDefinition('x', ArgType.INTEGER, True, 'x'),
            ArgDefinition('y', ArgType.INTEGER, True, 'y'),
        ],  # Args
        1,  # Max Concurrency
        'addition', # Description
        'awesome_addition' )  # custom endpoint name   
```
 * http://localhost:8888/awesome_addition

  
But, registering same python function with different endpoint name is not work properly because the function has same name attribute, *function.\_\_name\_\_* .  
I will fix it.

**Comment by yszkst** [comment]
resolved #36 .
